app/src/main/java/org/simple/clinic/search/results/PatientSearchResultsController.kt

```diff
@@ -37,28 +39,17 @@ class PatientSearchResultsController @Inject constructor(
   private fun showComputedAge(events: Observable<UiEvent>): Observable<UiChange> {
     return events.ofType<PatientSearchResultsScreenCreated>()
         .map { it.key }
-        .map { (_, age, dob) ->
-          val computedAge = when {
-            age.isNotBlank() -> age.trim().toInt()
-            else -> calculateAgeFromDateOfBirth(dob)
-          }
-          computedAge
-        }
+        .map { (_, age, dob) -> ageOrAgeFromDateOfBirth(age, dob) }
```

Comment:
This method naming seems weird. Might I suggest `coerceAgeFrom()` or `obtainAgeFrom()` instead?